src/lib/awsElasticSearch/index.js

```diff
@@ -163,15 +163,38 @@ const search = async (indexName, fields, query, passedClient, overrideBatchSize)
     let res;
     let searchAfter;
 
-    // Create search body.
-    let body = {
-      size: batchSize,
-      query: {
+    // Create query section.
+    // ReadMe:
+    // If we have more than one word (term) then use phrase matching
+    // If we have only one word (term) use query string with wildcards.
+    // Site Ref: https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/
+    const queryBody = query.trim().split(' ').length <= 1
+      ? {
         query_string: {
           query: `*${query}*`,
           fields,
         },
-      },
+      }
+      : {
+        bool: {
+          should: [
+            { match_phrase: { context: { slop: 0, query } } },
+            { match_phrase: { nonECLKCResources: { slop: 0, query } } },
+            { match_phrase: { ECLKCResources: { slop: 0, query } } },
+            { match_phrase: { recipientNextSteps: { slop: 0, query } } },
+            { match_phrase: { specialistNextSteps: { slop: 0, query } } },
+            { match_phrase: { activityReportGoals: { slop: 0, query } } },
+            { match_phrase: { activityReportObjectives: { slop: 0, query } } },
+            { match_phrase: { activityReportObjectivesTTA: { slop: 0, query } } },
+            { match_phrase: { activityReportObjectiveResources: { slop: 0, query } } },
```

Comment:
these should be added
activityReport.additionalNotes
goal/activityReportGoal.timeframe

Reply:
Yea we will add them as we go.

FYI this PR is being replaced by put-back-search-after.